python/hail/keytable.py

```diff
@@ -567,16 +567,54 @@ def select(self, column_names):
         >>> kt_result = kt1.select([])
 
         :param column_names: List of columns to be selected.
-        :type: list of str
+        :type: str or list of str
 
         :return: Key table with selected columns.
         :rtype: :class:`.KeyTable`
         """
-
+        if isinstance(column_names, list):
+            for c in column_names:
+                if not isinstance(c, str) and not isinstance(c, unicode):
+                    raise TypeError("expected str or unicode elements of 'column_names' list, but found %s" % type(c))
+        elif not isinstance(column_names, str) and not isinstance(column_names, unicode):
+            raise TypeError("expected str or list of str for parameter 'column_names', but found %s" % type(column_names))
+
+        column_names = wrap_to_list(column_names)
         new_key = [k for k in self.key if k in column_names]
         return KeyTable(self.hc, self._jkt.select(column_names, new_key))
 
     @handle_py4j
+    @typecheck_method(column_names=oneof(strlike, listof(strlike)))
+    def drop(self, column_names):
```

Comment:
Awesome. I think that actually this should take varargs, but that's true of a bunch of methods and it's better to be consistent with the other things in the 0.1 interface.